excel/src/main/java/org/apache/metamodel/excel/ExcelConfiguration.java

```diff
@@ -102,17 +115,33 @@ public boolean isSkipEmptyColumns() {
 		return skipEmptyColumns;
 	}
 
+    /**
+     * Defines if columns in the excel spreadsheet should be validated on datatypes while
+     * reading the spreadsheet.
+     * 
+     * @return a boolean indicating whether or not to validate column types.
+     */
+    public boolean isDetectColumnTypes() {
+        return detectColumnTypes;
+    }
+
 	@Override
 	protected void decorateIdentity(List<Object> identifiers) {
 		identifiers.add(columnNameLineNumber);
 		identifiers.add(skipEmptyLines);
 		identifiers.add(skipEmptyColumns);
+		identifiers.add(detectColumnTypes);
 	}
 
 	@Override
 	public String toString() {
 		return "ExcelConfiguration[columnNameLineNumber="
 				+ columnNameLineNumber + ", skipEmptyLines=" + skipEmptyLines
-				+ ", skipEmptyColumns=" + skipEmptyColumns + "]";
+				+ ", skipEmptyColumns=" + skipEmptyColumns +", detectColumnTypes="
+				+ detectColumnTypes + "]";
+	}
+
+	public int getEagerness() {
```

Comment:
Can you rename `getEagerness` to `getNumberOfLinesToScan`?

Reply:
Done with next commit.